### PR TITLE
Devop/update maker lib

### PR DESCRIPTION
**Pull request by SteveMieskoski**
### Devop

- [x] Updated CHANGELOG.md
- [x] Add PR label
- [x] No hard-coded strings

### CODE REVIEW

File: src/dapps/MakerDai/MakerDai.vue [hunk 1]
```diff
@@ -559,12 +559,12 @@ export default {
       this.openMoveModal = false;
     },
 
-    async migrateCdp(cdpId) {
-      const currentProxy = await this.getProxy();
-      if (currentProxy) {
-        await this._cdpService.give(cdpId, currentProxy);
-      }
-    },
+    // async migrateCdp(cdpId) {
```
Comment (gamalielhere) [inline src/dapps/MakerDai/MakerDai.vue:562]: Remove commented code

File: src/dapps/MakerDai/containers/CreateCDP/CreateCDP.vue [hunk 2]
```diff
@@ -229,14 +229,14 @@ export default {
...
     },
-    liquidationPenalty: {
-      type: BigNumber,
-      default: toBigNumber(0)
-    },
-    stabilityFee: {
-      type: BigNumber,
-      default: toBigNumber(0)
-    },
+    // liquidationPenalty: {
```
Comment (gamalielhere) [inline src/dapps/MakerDai/containers/CreateCDP/CreateCDP.vue:232]: commented code

File: src/dapps/MakerDai/containers/ManageCDP/ManageCDP.vue [hunk 3]
```diff
@@ -274,36 +274,36 @@ export default {
...
     },
-    liquidationPenalty: {
-      type: BigNumber,
-      default: toBigNumber(0)
-    },
-    stabilityFee: {
-      type: BigNumber,
-      default: toBigNumber(0)
-    },
+    // liquidationPenalty: {
```
Comment (gamalielhere) [inline src/dapps/MakerDai/containers/ManageCDP/ManageCDP.vue:277]: Commented code

File: src/dapps/MakerDai/makerHelpers/locateCdps.js [hunk 4]
```diff
@@ -27,37 +27,37 @@ async function locateCdps(self) {
...
-    withType: cdpIdToTypeMapping,
-    withProxy: cdps.map(entry =>
-      typeof entry !== 'number' ? entry.id : entry
-    ),
-    withoutProxy: cdpsWithoutProxy.map(entry =>
-      typeof entry !== 'number' ? entry.id : entry
-    )
-  };
-}
+// async function locateOldCdps(self, _cdpService) {
```
Comment (gamalielhere) [inline src/dapps/MakerDai/makerHelpers/locateCdps.js:30]: commented code

**Comment by gesseekur** [comment]
can you fix build so I can test :D

File: package.json [hunk 5]
```diff
@@ -132,6 +132,7 @@
     "marked": "1.1.0",
     "mew-maker-plugin": "0.0.2",
     "multicoin-address-validator": "git+https://github.com/MyEtherWallet/multicoin-address-validator.git",
+    "node-fetch": "2.6.0",
```
Comment (gamalielhere) [inline package.json:135]: Unused package

**Pull request merged**